client/components/LocalCompanies/BussinessCard.jsx

```diff
@@ -42,7 +34,11 @@ const BussinessCard = ({ data }) => {
 
     //* * axios to update smoves with fav companies * *//
     axios.patch(`/user/${userData.email}`, {data: userData.smoves})
-      .then((data) => console.log('FavCompany successfully updated: ', data))
+      .then(({ data }) => {
+        console.log('FavCompany successfully updated: ', data);
```

Comment:
Do you want to keep this console.log?

Reply:
thanks.  i fixed it.  stupid console.logs lol